src/core/pointcloud/qgspointcloudlayerrenderer.h

```diff
@@ -91,7 +91,7 @@ class CORE_EXPORT QgsPointCloudLayerRenderer: public QgsMapLayerRenderer
     QgsPointCloudAttributeCollection mAttributes;
     QgsGeometry mCloudExtent;
     QList< QgsMapClippingRegion > mClippingRegions;
-    const QVector< QgsPointCloudSubIndex > mSubIndexes;
+    const QList<QgsPointCloudSubIndex *> mSubIndexes;
```

Comment:
Isn't this making 2d rendering even **less** thread safe? 😱

Reply:
Reverted